src/commands/config.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const AppConfig = require('../models/app_configuration.js');
+const Application = require('../models/application.js');
+const Logger = require('../logger.js');
+
+const CONFIG_KEYS = [
+  { id: 'name', name: 'name', displayName: 'Name' },
+  { id: 'description', name: 'description', displayName: 'Description' },
+  { id: 'zero-downtime', name: 'homogeneous', displayName: 'Zero-downtime deployment', displayer: zeroDowntimeDisplayer },
+  { id: 'sticky-sessions', name: 'stickySessions', displayName: 'Sticky sessions', displayer: booleanDisplayer },
+  { id: 'cancel-on-push', name: 'cancelOnPush', displayName: 'Cancel current deployment on push', displayer: booleanDisplayer },
+];
+
+function getConfigById (id) {
+  const [config] = CONFIG_KEYS.filter((config) => config.id === id);
+  if (config === undefined) {
+    Logger.error(`Invalid configuration name: ${id}.`);
+    Logger.error(`Available configuration names are: ${CONFIG_KEYS.map((config) => config.id).join(', ')}.`);
+  }
+  return config;
+}
+
+function zeroDowntimeDisplayer (homogeneous) {
+  return booleanDisplayer(!homogeneous);
+}
+
+function booleanDisplayer (value) {
+  return (value) ? 'enabled' : 'disabled';
+}
+
+function defaultDisplayer (value) {
+  return `${value}`;
```

Comment:
Is it written this way to cast it to a string?

Reply:
Sort of. At first in some other version of the code it was to have different "symmetrical" branches, and I figured it doesn't hurt, we we can just use identity here in fact